### DISCUSSION TITLE
Combine different net tasks

### DISCUSSION THREAD

**Discussion by klausinfo** [ultralytics/yolov5#3023] [Q&A]
Let us say I want to read car plates. A good idea could be to train three different nets: for cars, plates and numbers. In order to be fast, use the first net to detect cars, than insinde the car bounding box look for the plate. Inside this plate bounding box look for the numbers.

But is this really the fastest way? It takes the time of 3 nets. Or is there a way to combine all 3 nets in one?

What is the best way to solve this? Any experiences would be a great help. Thanks.

**Comment by glenn-jocher** [comment]
@klausinfo sure, cascaded detection workflows may be beneficial in some circumstances. Only you know what's best for your specific use case.

**Reply by klausinfo** [reply]
@glenn-jocher Ok, cascaded detection workflows is a typicall way to solve this. I didn't know - it was just an idea. Thanks for the help and fast response. And thanks a lot for your great work here :))
